storage/simulation/script.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Bram Gruneir (bram+code@cockroachlabs.com)
+
+package main
+
+// Operation enumerates the possible scripted operations that can occur to a
+// cluster.
+type Operation int
+
+// These are the possible scripted operations.
+const (
+	_ Operation = iota
+	OpSplitRangeRandom
```

Comment:
why not make this an enumeration of strings? seems like you could map strings directly to your storage format. this is why i think that format should be in this PR.